docs/introduction.md

```diff
@@ -59,3 +59,9 @@ After the promise is settled, a second action will be dispatched. If the the pro
   }
 }
 ```
+
+The default promise status suffix can be imported from this module.
+
+```js
+import { PENDING, FULFILLED, REJECTED } from 'redux-promise-middleware'
```

Comment:
Ah, thanks for catching this @tomatau.

I'd agree this is verbose. 

You should be able to solve this by renaming the exports:

```js
export { DEFAULT_PENDING as PENDING, DEFAULT_FULFILLED as FULFILLED, ...  };
```

The other option would be to avoid renaming the exports and just rename the constants.